GDJS/Runtime/runtimeobject.ts

```diff
@@ -318,7 +318,10 @@ namespace gdjs {
       this.layer = '';
       this._livingOnScene = true;
       //@ts-ignore Reinitialize is like a constructor, it can overwrite the readonly property.
-      this.id = runtimeScene.createNewUniqueId();
+      this.id = runtimeScene
+        //
+        .getScene()
+        .createNewUniqueId();
```

Comment:
That doesn't seem like a proper fix... If `createNewUniqueId`is not valid to call on a `gdjs.RuntimeInstanceContainer`, TypeScript should be catching it? 🤨 And that error could happen again too.
Shouldn't this method either be removed from gdjs.RuntimeInstanceContainer, or implemented there to call `this.getScene().createNewUniqueId()`?

Reply:
There is a `@ts-ignore` above. This is why I broke the line with an empty comment.